### DISCUSSION TITLE
How to get the status of a path ?

### DISCUSSION THREAD

**Discussion by Canop** [GitoxideLabs/gitoxide#862] [Q&A] [answered]
Starting from a repository,
how do you get the git status of a path (i.e. the equivalent of [git2's Status](https://docs.rs/git2/latest/git2/struct.Status.html)) ? 
How do you get the list of all statuses ?

This is probably simple and obvious, so I'm sorry, but I couldn't find it in the doc.

**Comment by Byron** [comment, marked as answer]
At the time of writing, this isn't implemented yet, but some progress has been made.

You can track progress by checking in on [this tracking issue](https://github.com/Byron/gitoxide/issues/301) (*see the `Tasks for status` section*). I'd expect it to be done within this year for sure, but hopefully in a couple of months.
